tests/countries/test_bonaire_sint_eustatius_and_saba.py

```diff
@@ -0,0 +1,314 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.bonaire_sint_eustatius_and_saba import BonaireSintEustatiusAndSaba, BQ, BES
+from tests.common import CommonCountryTests
+
+
+class TestBonaireSintEustatiusAndSaba(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(2011, 2050)
+        super().setUpClass(BonaireSintEustatiusAndSaba, years=years)
+        cls.subdiv_holidays = {
+            subdiv: BonaireSintEustatiusAndSaba(subdiv=subdiv, years=years)
+            for subdiv in BonaireSintEustatiusAndSaba.subdivisions
+        }
+
+    def test_country_aliases(self):
+        self.assertAliases(BonaireSintEustatiusAndSaba, BQ, BES)
+
+    def test_new_years(self):
+        name = "Nieuwjaarsdag"
+        self.assertHolidayName(name, (f"{year}-01-01" for year in range(2011, 2050)))
```

Comment:
```suggestion
    def test_new_years_day(self):
        self.assertHolidayName("Nieuwjaarsdag", (f"{year}-01-01" for year in range(2011, 2050)))
```
And the same below for similar holidays.